### DISCUSSION TITLE
Elastic Fleet Standalone -Security Onion 2.4

### DISCUSSION THREAD

**Discussion by camdg12** [Security-Onion-Solutions/securityonion#12673] [Unsupported Versions]
Hello 
We have a distributed architecture of Security Onion
It is possible to integrate a Standalone Elastic Fleet  without using the SO ISO containing the Fleet Role ? 
Best Regards !

**Comment by reyesj2** [comment]
Yes, you can do a network install of the stand alone fleet node without using the Security Onion ISO.  https://docs.securityonion.net/en/2.4/network-installation.html#network-installation

**Comment by camdg12** [comment]
Hello !
Thanks for your assistance, but we would like to use an external fleet instead of using SO (ISO or network installation).
